biz.aQute.bndlib/src/aQute/bnd/build/Workspace.java

```diff
@@ -1521,16 +1548,29 @@ Repository getResourceRepository() throws Exception {
 
 	/**
 	 * A macro that returns a set of resources in bundle selection format from
-	 * the repository. For example:
+	 * the repositories. For example:
 	 *
 	 * <pre>
 	 * ${findproviders;osgi.wiring.package;(osgi.wiring.package=aQute.bnd.build)}
 	 * </pre>
 	 */
 
 	public String _findproviders(String[] args) throws Exception {
-		Macro.verifyCommand(args, _findprovidersHelp, null, 2, 3);
-		return findProviders(args[1], args.length > 2 ? args[2] : null).map(Capability::getResource)
+		Macro.verifyCommand(args, _findprovidersHelp, null, 2, 4);
+		return findProvidersMacro(args);
+	}
+
+	private enum STRATEGY {
```

Comment:
Perhaps we should call the enum class `ResourceRepositoryStrategy`? Since there are many things for which we want a strategy.